### DISCUSSION TITLE
automatically receive csv files from bank using playwright

### DISCUSSION THREAD

**Discussion by nebula2** [firefly-iii/firefly-iii#4768] [General]
Hey :)
me and my cousin are using firefly for round about a year now. we both have more than 5 accounts each. updading firefly through the csv importer is a blessing. but receiving those files from your bank is not that comfortable.

After discussing possibilities we thought about using playwright to receive the files.

I managed to get this working. I built a rest api which i can pass a config identifier and a date. i receive the csv file along with a video recording of playwright generating this.

I do however not feel confident with sharing code at the moment as this is by nature a very sensitive thing. one mistake and you could get kicked out of your account.

Long story short: it is possible. is it good? idk xD

Just wanted to share
